usaspending_api/awards/v2/filters/location_filter_geocode.py

```diff
@@ -249,7 +257,9 @@ def build_es_city_query(query: object, bool_should: bool,
     return query
```

Comment:
(I can't add this comment to line 247|255 so adding here:
Need to add .upper() 
`query["bool"]["must"].append({"match": {"{}_state_code".format(scope): es_sanitize(state_code).upper()}})`